### PR TITLE
Get years months date hours min sec msec functions

### PR DESCRIPTION
**Pull request by ciorg**
functions:
  * getMilliseconds
  * getSeconds
  * getMinutes
  * getHours
  * getDate
  * getMonth
  * getYear

Question on getDate - that could be confused with a complete date, not day of the month.  Maybe it should be getDayOfMonth?

### CODE REVIEW

File: packages/data-mate/src/function-configs/date/getHours.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            output: 10
+        },
+        {
+            args: {},
+            config: {
+                version: 1,
+                fields: { testField: { type: FieldType.String } }
+            },
+            field: 'testField',
+            input: '05/22/2021 EST',
```
Comment (peterdemartini) [inline packages/data-mate/src/function-configs/date/getHours.ts:41]: How does this work without specifying the format?

Comment (ciorg) [reply packages/data-mate/src/function-configs/date/getHours.ts:41]: new Date('string') can still parse that, so the getValidDate function is able to handle that format.

Comment (peterdemartini) [reply packages/data-mate/src/function-configs/date/getHours.ts:41]: Gotcha.

**Review by peterdemartini — APPROVED** [review]
LGTM

**Review by jsnoble — APPROVED** [review]
LGTM

**Pull request merged**